db/migrate/20160208165815_create_events.rb

```diff
@@ -0,0 +1,12 @@
+class CreateEvents < ActiveRecord::Migration
+  def change
+    create_table :events do |t|
+      t.timestamp :datetime
```

Comment:
Should I create new migrations to delete that column and create new with the right type?